### ISSUE TITLE
Add notification via POST

### ISSUE THREAD

**Issue by felipecassiors** [TheConnMan/docker-hub-rss#13]
How about that:

A option to send a POST notification to an endpoint every time a tag get updated. This would make us able to trigger a Docker Automated Build (in Docker Hub) using the external trigger. 

![image](https://user-images.githubusercontent.com/29582865/74685451-60983000-51ad-11ea-86dc-eea030e03601.png)

**Comment by TheConnMan** [comment]
I can see the value in doing that. I'd really like to not introduce a database to this project. Could we use a JSON config file to map Docker Hub repositories to an array of URLs to POST to on update?

**Comment by felipecrs** [comment]
Good idea in order to skip a database. That would work, as long as the file isn't exposed by the frontend we're fine. Anyone with the URL would be able to trigger a Docker Automated Build.

**Comment by TheConnMan** [comment]
Agreed. Do you have a recommended POST body payload? We'll need a documented API so users can pull info from the payload.

**Comment by felipecrs** [comment]
Actually, Docker Hub doesn't require a payload, the only requirement is to send a POST request in that URL. However, we have already [MicroBadger](https://microbadger.com/notifications) trying to do something more or less the same:

![image](https://user-images.githubusercontent.com/29582865/74840985-ac122180-5306-11ea-8025-e8efb5417eec.png)

**Comment by TheConnMan** [comment]
I see. I'm fine with initially no POST body.

Is this something you could take a crack at? I'm happy to comment on a PR and provide guidance.

**Comment by TheConnMan** [comment]
@felipecassiors after some more thinking I don't think this feature will work. The beauty of RSS is that there's not state in this project. It just takes tags and converts them into an RSS feed. There's no concept of "new" tags. It's the RSS reader's job to determine what is new and old so we don't have a trigger.

What do you think of using something like https://ifttt.com/feed to read this RSS feed and call out to some webhook (https://ifttt.com/maker_webhooks) when a new item is available?

**Comment by JuniorJPDJ** [comment]
The separate service would be better as it could be used more general.
Eg. you could trigger rebuild on change of dh tag OR change of git tag OR other event ;)
Everything by RSS and using this bridge as DH provider
TL;DR: keep it simple, stupid; build a bigger stack with usage of puzzles, not one big unmaintainable app

**Comment by TheConnMan** [comment]
I like it. In that case we're closing this as Won't Do, correct?
